code/datums/jobs/access.dm

```diff
@@ -219,6 +219,8 @@
 			. = size ? "Dr. " : "Doctor"
 		if("CCMO")
 			. = size ? "Prof. " : "Professor"
+		if ("CMN")
```

Comment:
```suggestion
		if("CMN")
```
Whitespace is usually not enforced but consistency is important, and the rest of the lines use this style

Reply:
I see! I'll keep an eye out for that the next time i'll PR something